lib/babe/verify_integration_test.go

```diff
@@ -241,33 +211,105 @@ func TestVerificationManager_VerifyBlock_MultipleEpochs(t *testing.T) {
 	epochData, err := babeService.initiateEpoch(0)
 	require.NoError(t, err)
 
-	futureEpoch := uint64(5)
+	slot := getSlot(t, runtime, time.Unix(6, 0))
+	block := createTestBlockWithSlot(t, babeService, &genesisHeader, [][]byte{}, 0, epochData, slot)
+
+	err = vm.VerifyBlock(&block.Header)
+	require.NoError(t, err)
+}
+
+func TestVerificationManager_VerifyBlock_FutureEpoch(t *testing.T) {
```

Comment:
Since we are testing a case of multiple epochs, I wanted to have an isolated test where we only test future epoch